node_modules/oae-authentication/lib/api.js

```diff
@@ -87,7 +87,6 @@ var _configUpdate = function(tenantAlias) {
     }
 };
 
-ConfigAPI.on('cleared', _configUpdate);
```

Comment:
Also, what do some strategies being configured globally have to do with this particular event?